engines/dfc_provider/config/routes.rb

```diff
@@ -11,6 +11,9 @@
   resources :enterprise_groups, only: [:index, :show] do
     resources :affiliated_by, only: [:create, :destroy], module: 'enterprise_groups'
   end
-  resources :persons, only: [:show]
-  resources :anonymous_orders, only: [:index]
+  resources :persons, only: [:show] do
+    member do
+      get :affiliate_sales_data
+    end
+  end
```

Comment:
I think we agreed the Person would be the researcher gaining access to the data (by being affiliated to all the Enterprises that have agreed to share data)

Reply:
Initially I was working in a separate controller but since the [specs](https://app.swaggerhub.com/apis-docs/RAGGEDSTAFF23/DFC-INRAE-Endpoint/v0.1#/INRAE/get_api_dfc_Persons__id__AffilateSalesData) have this data under a `Person` resource I just followed the url structure and rails convention. But I can keep the url and point it to a `AffiliateSalesController` if you think we should separate the concerns.

Reply:
Yes, I think it would be good to separate this code.